src/components/Menu/MenuLinkList.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import Analytics from '../../Analytics';
+
+function MenuLinkList() {
```

Comment:
Menu name is redundant as it is 'packed' inside the Menu dir, same at MenuSocialProfile